dom-querying/main.js

```diff
@@ -0,0 +1,14 @@
+console.log('hello, world');
+
+console.log(document.querySelector('h1'));
+console.dir('h1');
```

Comment:
this is not what you want. You need to pass a DOM element to dir. I recommend storing the queryselected element in a variable so you can pass that variable to both the log and dir method calls.